### PR TITLE
Add timeout and correct k8s config sample

### LINKED ISSUE #55: Integration stalls when a discovered agent does not respond
Originally reported here: https://newrelic.slack.com/archives/CP2EWSX7C/p1647964852438629

> The problem has to do with timeouts and the way the OHI fan out is structured. With HCP the leader does not expose the API but we keep making requests that are effectively serial and eventually the Infra Agent gives up on the OHI. I hacked in a bit of code that skips an Agent (in the OHI's sense) after the first network error and that fixes the issue- the OHI reports.

A patch for this can be found in https://github.com/msummers-nr/nri-consul/commit/3678b4a329ccea0ac1c0fd52ab189ce64789f73e

### PR DESCRIPTION
**Pull request by alvarocabanas**
Fixes #55

### CODE REVIEW

File: src/args/argument_list.go [hunk 1]
```diff
@@ -15,6 +17,7 @@ type ArgumentList struct {
 	Hostname               string `default:"localhost" help:"The agent node Hostname or IP address to connect to"`
 	Port                   string `default:"8500" help:"Port to connect to agent node"`
 	Token                  string `default:"" help:"ACL Token if token authentication is enabled"`
+	Timeout                int    `default:"0" help:"Timeout for an API call"`
```
Comment (gsanchezgavier) [inline src/args/argument_list.go:20]: would make sense to use the golang time syntax here as is used in `interval` or `timeout` ? i.e: `30s`

File: src/args/argument_list_test.go [hunk 2]
```diff
@@ -81,27 +89,25 @@ func Test_ArgumentList_CreateAPIConfig(t *testing.T) {
 				Port:                   "8500",
 				Token:                  "my_token",
 				EnableSSL:              true,
-				TrustServerCertificate: false,
-				CABundleDir:            "ca_dir",
-				CABundleFile:           "ca_file",
+				TrustServerCertificate: true,
```
Comment (gsanchezgavier) [inline src/args/argument_list_test.go:92]: would make sense to keep this case and add an extra with this `TrustServerCertificate: true` ?

File: src/args/argument_list.go (hunk 1 above)
Comment (alvarocabanas) [reply src/args/argument_list.go]: I have had to add it to all the unit tests, because the zero value of string is "" and the default we set is only honored when integration.new is called.

File: src/args/argument_list.go [hunk 3]
```diff
@@ -15,6 +17,7 @@ type ArgumentList struct {
 	Hostname               string `default:"localhost" help:"The agent node Hostname or IP address to connect to"`
 	Port                   string `default:"8500" help:"Port to connect to agent node"`
 	Token                  string `default:"" help:"ACL Token if token authentication is enabled"`
+	Timeout                string `default:"0s" help:"Timeout for an API call"`
```
Comment (gsanchezgavier) [inline src/args/argument_list.go:20]: we might consider to add a default timeout , perhaps related to the default integration interval 30s
